jobis-infrastructure/src/main/java/team/retum/jobis/event/application/ApplicationMessage.java

```diff
@@ -0,0 +1,71 @@
+package team.retum.jobis.event.application;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import team.retum.jobis.domain.application.event.ApplicationsStatusChangedEvent;
+
+@Getter
+@AllArgsConstructor
+public class ApplicationMessage {
+    private final String title;
+    private final String content;
+
+    public static ApplicationMessage from(ApplicationsStatusChangedEvent event, String companyName) {
```

Comment:
from은 보통 기본 객체를 사용해서 새로운 객체를 만들때 사용하는 메서드에 사용됩니다. 그래서 보통 매개변수의 개수가 하나입니다. 이 경우에는 여러 인자를 받아서 새 객체를 생성하니 of를 사용하는게 좋아보여요

```suggestion
    public static ApplicationMessage of(ApplicationsStatusChangedEvent event, String companyName) {
```